packages/server/postgres/utils/checkTeamEq.ts

```diff
@@ -1,17 +1,19 @@
-import Team from '../../database/types/Team'
 import getRethink from '../../database/rethinkDriver'
-import {checkTableEq} from './checkEqBase'
+import Team from '../../database/types/Team'
 import getTeamsByIds from '../queries/getTeamsByIds'
+import {checkTableEq} from './checkEqBase'
 
-const alwaysDefinedFields: (keyof Partial<Team>)[] = [
+const alwaysDefinedFields: (keyof Team)[] = [
   'name',
   'createdAt',
   'isArchived',
   'isPaid',
   'tier',
   'orgId',
-  'updatedAt'
+  'createdBy',
```

Comment:
Nice caught!
I suppose we should skip this field as well. Because, as far as I understand, the `getPairNeFields` logic only works in cases where the value from RethinkDB can be `null` or `undefined`.